template/logEntry/logEntry.proto

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package logEntry;
+
+// TODO: STUFF THAT IS COMMENTED OUT NEEDS TO BE TURNED ON ONCE CODE GEN IS FIXED
```

Comment:
What's broken about it today? I'm not convinced we can populate any of these fields at runtime via codegen'd stuff. In particular:
- I don't know how we could produce a string value "log_name" at runtime in code genned code: where would the values come from?
- By the same token, how can we handle severity?
- What do we stick in the log_template string? Again, it's just a string from codegen's perspective, not special, so I don't see how we could ever provide a meaningful value.

These are all the things I thought we'd discussed moving out of templates. The rule of thumb we'd agreed on is that only runtime values belong in templates. Look at the metric template: it _only_ has ValueType fields, all of the non-runtime values (the type of the metric, buckets, etc) are in other configuration (today the aspect, in the future it'll move out to individual adapters).